test/zmumu_pp5tev/tp_from_aod_Data.py

```diff
@@ -0,0 +1,629 @@
+      HLT_HIL2Mu5_NHitQ10 = cms.string("!triggerObjectMatchesByPath('HLT_HIL2Mu5_NHitQ10_v*',1,0).empty()"),
+      HLT_HIL3Mu5 = cms.string("!triggerObjectMatchesByPath('HLT_HIL3Mu5_v*',1,0).empty()"),
+      HLT_HIL3Mu5_NHitQ10 = cms.string("!triggerObjectMatchesByPath('HLT_HIL3Mu5_NHitQ10_v*',1,0).empty()"),
+      HLT_HIL2Mu7 = cms.string("!triggerObjectMatchesByPath('HLT_HIL2Mu7_v*',1,0).empty()"),
+      HLT_HIL3Mu7 = cms.string("!triggerObjectMatchesByPath('HLT_HIL3Mu7_v*',1,0).empty()"),
+      )
+HighPtSingleTriggerFlags = cms.PSet(
+      # Single Muon Trigger Paths
+      HLT_HIL2Mu12 = cms.string("!triggerObjectMatchesByPath('HLT_HIL2Mu12_v*',1,0).empty()"),
+      HLT_HIL3Mu12 = cms.string("!triggerObjectMatchesByPath('HLT_HIL3Mu12_v*',1,0).empty()"),
+      HLT_HIL2Mu15 = cms.string("!triggerObjectMatchesByPath('HLT_HIL2Mu15_v*',1,0).empty()"),
+      HLT_HIL3Mu15 = cms.string("!triggerObjectMatchesByPath('HLT_HIL3Mu15_v*',1,0).empty()"),
+      HLT_HIMu15 = cms.string("!triggerObjectMatchesByPath('HLT_HIMu15_v*',1,0).empty()"),
+      HLT_HIMu17 = cms.string("!triggerObjectMatchesByPath('HLT_HIMu17_v*',1,0).empty()"),
+      HLT_HIL2Mu20 = cms.string("!triggerObjectMatchesByPath('HLT_HIL2Mu20_v*',1,0).empty()"),
+      HLT_HIL3Mu20 = cms.string("!triggerObjectMatchesByPath('HLT_HIL3Mu20_v*',1,0).empty()"),
+      )
+### Tracking
+TRACK_CUTS = "track.isNonnull && track.hitPattern.trackerLayersWithMeasurement > 5 && track.hitPattern.pixelLayersWithMeasurement > 0"
+
+
+process.tagMuons = cms.EDFilter("PATMuonSelector",
+    src = cms.InputTag("patMuonsWithTrigger"),
+    cut = cms.string("pt > 15 && abs(eta)<2.4 && "+MuonIDFlags.TightId.value()+" && ("
+       +    "!triggerObjectMatchesByPath('HLT_HIL2Mu3_NHitQ10_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu3_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu3_NHitQ10_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL2Mu5_NHitQ10_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu5_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu5_NHitQ10_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL2Mu7_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu7_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL2Mu12_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu12_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL2Mu15_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu15_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIMu15_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIMu17_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL2Mu20_v*',1,0).empty()"
+       + "|| !triggerObjectMatchesByPath('HLT_HIL3Mu20_v*',1,0).empty())"
+       + " && pfIsolationR04().sumChargedHadronPt/pt < 0.2"),
```

Comment:
Should we also require isolation for low pT muon tags? (e.g for JPsi)

Reply:
yes, not sure about iso. Let's remove it, and again apply it in the fitting step if needed